Mathlib/AlgebraicGeometry/EllipticCurve/DivisionPolynomial/Basic.lean

```diff
@@ -179,16 +179,16 @@ lemma preΨ'_three : W.preΨ' 3 = W.Ψ₃ :=
 lemma preΨ'_four : W.preΨ' 4 = W.preΨ₄ :=
   preNormEDS'_four ..
 
-lemma preΨ'_odd (m : ℕ) : W.preΨ' (2 * (m + 2) + 1) =
-    W.preΨ' (m + 4) * W.preΨ' (m + 2) ^ 3 * (if Even m then W.Ψ₂Sq ^ 2 else 1) -
-      W.preΨ' (m + 1) * W.preΨ' (m + 3) ^ 3 * (if Even m then 1 else W.Ψ₂Sq ^ 2) :=
-  preNormEDS'_odd ..
-
 lemma preΨ'_even (m : ℕ) : W.preΨ' (2 * (m + 3)) =
```

Comment:
Can you please fix in the docstring that tilde seems now to be a prime?

Reply:
The tilde correspond to the `pre`, which I have made slightly clearer now. The primed versions are just the unprimed versions but defined over N rather than over Z. The linter does not complain for `preΨ'` because there is already a docstring, and for the lemmas `preΨ'_*` because the primes don't appear at the end? What should I do here?

Reply:
I think that the solution you propose below (appending `_ofNat`) is the best.

Reply:
I can't call them `ofNat`, because that's the name of a lemma below...

Reply:
Well, can you come up with another name indicating that it is `Nat`-related?